core/src/main/java/org/akaza/openclinica/bean/managestudy/EventDefinitionCRFBean.java

```diff
@@ -579,38 +577,38 @@ public void setEventName(String eventName) {
         this.eventName = eventName;
     }
 
-	public CRFVersionBean getDefaultCRF() {
-		return defaultCRF;
-	}
+    public FormLayoutBean getDefaultCRF() {
+        return defaultCRF;
+    }
+
+    public void setDefaultCRF(FormLayoutBean defaultCRF) {
+        this.defaultCRF = defaultCRF;
+    }
 
-	public void setDefaultCRF(CRFVersionBean defaultCRF) {
-		this.defaultCRF = defaultCRF;
-	}
+    public boolean isParticipantForm() {
+        return participantForm == true;
```

Comment:
This was done by Ron back in 2015 . This is his pull request https://github.com/OpenClinica/OpenClinica/commit/6e5bff933200e0e9c8d00c84eb46f97e10f9983f
Don't know what was the rational behind it.